chunjun-connectors/chunjun-connector-starrocks/src/main/java/com/dtstack/chunjun/connector/starrocks/sink/StarRocksOutputFormat.java

```diff
@@ -51,6 +51,11 @@ protected void openInternal(int taskNumber, int numTasks) {
                         .collect(Collectors.toList());
 
         streamLoadManager = new StreamLoadManager(starRocksConfig);
+        if (!streamLoadManager.tableHasPartition()) {
+            throw new RuntimeException(
+                    "data cannot be inserted into table with empty partition. "
+                            + "Use `SHOW PARTITIONS FROM star_test` to see the currently partitions of this table");
```

Comment:
what  does `star_test` mean?

Reply:
。。。 My mistake。Should be the output table name